packages/google_maps_flutter/ios/google_maps_flutter.podspec

```diff
@@ -16,7 +16,7 @@ A new flutter plugin project.
   s.public_header_files = 'Classes/**/*.h'
   s.dependency 'Flutter'
   s.dependency 'GoogleMaps'
-  s.compiler_flags = '-fno-modules'
```

Comment:
With this change I'm able to use module imports in a Swift and Objective-C app (building with libraries and frameworks). @iskakaushik I believe that's enough to prove I didn't regress https://github.com/flutter/plugins/pull/1734?
```objective-c
@import GoogleMapsBase;
@import google_maps_flutter;
```